src/components/common/Problem.vue

```diff
@@ -1,10 +1,16 @@
 <template>
-	<div ref="renderDiv" v-html="html" />
+	<iframe
+		v-resize="{ checkOrigin: false }"
+		:srcdoc="html"
+		width="100%"
+		frameborder="0"
+	></iframe>
 </template>
 
 <script lang="ts">
-import type { Ref } from 'vue';
-import { defineComponent, ref, watch } from 'vue';
+import { defineComponent, Ref, ref, watch } from 'vue';
```

Comment:
Note that all typescript types should be imported separately from imported javascript objects with `import type` the way I set this before.  The type is then completely erased from the generated javascript code.  This is the case as of version 3.8 of typescript.  In the future eslint will complain about this not being done.

This should be done with all type imports throughout the code.

Although I should also add that rarely do you need to import the vue `Ref` type.  Instead the ref template should be used or for basic types just type inference.  Instead of `const html: Ref<string> = ref('');` this should simply be `const html = ref('');`.  Type inference will do the correct thing here. This is valid for string, boolean, and numeric values.  For more complex types use the ref template.  For example on line 70 of src/components/instructor/ClasslistManager.vue instead of `const selected: Ref<Array<User>> = ref([]);` this should be `const selected = ref<Array<User>>([]);`.  This is all in Vue's documentation.  See https://v3.vuejs.org/guide/typescript-support.html#typing-refs.

Reply:
Looks like I botched the merge conflict... 🤦

Reply:
No problem.  I really just wanted to take the opportunity to point out the correct `ref` usage for @pstaabp.

Reply:
Good for me to see too -- I wasn't aware of this: `const selected = ref<Array<User>>([]);`

Reply:
I didn't realize about the difference in importing types and js functions/objects.  I'll also move to this.